webrtc.html

```diff
@@ -1297,6 +1299,25 @@
               </li>
 
               <li>
+                <p>If <var>candidate</var> is <code>null</code>, then run the
+                following steps:</p>
+                <ol>
+                  <li>
+                    <p>For each media section in the last successfully applied
+                    remote description, perform the processing for an
+                    end-of-candidates indication for said media section as
+                    defined in [[TRICKLE-ICE]], unless an ICE restart occurred
+                    for said media section since setting the last remote
+                    description.</p>
+                  </li>
+                  <li>
+                    <p>Return a promise resolved with
+                    <code>undefined</code>.</p>
```

Comment:
Should it resolve immediately, or once the processing is done?  It seems like it shouldn't resolve until it's done.